src/main/java/com/feedbacksolution/model/Tweet.java

```diff
@@ -0,0 +1,211 @@
+package com.feedbacksolution.model;
```

Comment:
So we are good to merge ? Who will merge ? You or me ? How do I know if this request has been approved ?